### PR TITLE
For with formwith

### PR DESCRIPTION
**Pull request by ImMPrada**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,24 +1,34 @@
...
 
-* Deployment instructions
+- create the database
+`rails db:create`
 
-* ...
+- run migrations
+`rails db:migrate`
+
+- if you want you can popullate the database with seed
```
Comment (mapra99) [inline README.md]:
```suggestion
- if you want you can populate the database with seed
```

File: app/controllers/users_controller.rb [hunk 2]
```diff
@@ -1,19 +1,112 @@
...
+  end
+
+  def destroy
+    @user = User.find(user_id)
+
+    @user.destroy
+    respond_to do |format|
+      remove_user(format)
+      format.html { redirect_to users_path, notice: 'Users updated' }
+    end
```
Comment (mapra99) [inline app/controllers/users_controller.rb:47]: I think this is missing a way to handle errors, right? Something like
```ruby
if @user.destroy
  respond_to do |format|
      remove_user(format)
      format.html { redirect_to users_path, notice: 'Users updated' }
    end
else
  # respond with an error
end
```

File: app/views/users/index.erb
Comment (mapra99) [inline app/views/users/index.erb:1]: I think this file should be named `index.html.erb` the `html` is missing

**Review by mapra99 — APPROVED** [review]
cool cool! Left some minor comments

File: app/controllers/users_controller.rb (hunk 2 above)
Comment (ImMPrada) [reply app/controllers/users_controller.rb:47]: I'll work on this in another branch

**Pull request merged**